sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_receipts_async.py

```diff
@@ -57,11 +57,11 @@ async def analyze_receipts_async():
 
     for idx, receipt in enumerate(receipts.documents):
         print("--------Recognizing receipt #{}--------".format(idx + 1))
-        receipt_type = receipt.fields.get("ReceiptType")
+        receipt_type = receipt.doc_type
         if receipt_type:
             print(
-                "Receipt Type: {} has confidence: {}".format(
-                    receipt_type.value, receipt_type.confidence
+                "Receipt Type: {}".format(
+                    receipt_type.value
```

Comment:
Oh cool, didn't realize docType would replace ReceiptType like that. Should we simplify and remove the conditional?

`print("Receipt type: {}".format(receipt.doc_type or "N/A"))`

Reply:
I definitely want to confirm this with the rest of the crew, but the receipt schema doesnt include ReceiptType anymore, hence the switch to docType. Simplified the condition!